pkgs/applications/science/electronics/xyce/default.nix

```diff
@@ -156,8 +156,10 @@ stdenv.mkDerivation rec {
       "doc/Reference_Guide/Xyce_RG"
       "doc/Release_Notes/Release_Notes_${lib.versions.majorMinor version}/Release_Notes_${lib.versions.majorMinor version}")
 
-    # Release notes refer to an image not in the repo.
-    sed -i -E 's/\\includegraphics\[height=(0.5in)\]\{snllineblubrd\}/\\mbox\{\\rule\{0mm\}\{\1\}\}/' ''${docFiles[2]}.tex
+    # Release notes refer to 4 images not in the repo.
```

Comment:
Please link an upstream issue here

Reply:
There was a discussion about building the documentation here
https://groups.google.com/g/xyce-users/c/MxeViRo8CT4/m/ppCY7ePLEAAJ
The LaTeX class is not distributed outside of SANDIA. The images are organizations logo which are not distributed freely by organizations.

I'll put a link to the above link.